o/opentelemetry-php/build_info.json

```diff
@@ -2,14 +2,13 @@
     "package_name" : "opentelemetry-php",
     "github_url":"https://github.com/open-telemetry/opentelemetry-php",
     "version": "0.0.17",
-    "package_dir": "o/opentelemetry-php/latest",
+    "package_dir": "o/opentelemetry-php",
     "docker_build": false,
     "build_script" : "opentelemetry-php_ubi_8.5.sh",
-    
+
```

Comment:
@Pratikt2312 Same comments here. Please check

Reply:
Yes here also I checked the older build script version 0.0.12 on latest php version 8.0.25 its working well,so not need change.